packages/@azure/servicebus/data-plane/test/testUtils.ts

```diff
@@ -383,3 +384,18 @@ export async function purge(
     }
   }
 }
+
+// Maximum wait duration for the Streaming Receiver to receive the messages = `10000 ms`(10 seconds)(= maxWaitTime)
+// Keep checking whether the predicate is true after every `1000 ms`(1 second) (= delayBetweenRetries)
+export async function DelayStreaming(
+  predicate: () => boolean,
+  delayBetweenRetries: number = 1000,
```

Comment:
You might want to append `InMilliseconds` to the parameter names for clarity (so `delayBetweenRetriesInMilliseconds`).  But if it's a well-known convention in JavaScript that timespans are represented as integers of milliseconds, it may be redundant.